File: back/Authentication/Entities/CompanyUserAccount.cs

namespace Authentication.Entities;


public class CompanyUserAccount
{
    public int CompanyAuthId { get; set; }
    public CompanyAuth? CompanyAuth { get; set; }

    public int UserAccountId { get; set; }
    public UserAccount? UserAccount { get; set; }

    public DateTime LinkedOn { get; set; } = DateTime.UtcNow;
}

File: front/src/components/authentication/dtos/company-user-account.ts
import { CompanyAuth } from "./company-auth";
import { UserAccountAuthDto } from "../../authentication/dtos/user-account-auth-dto";

export class CompanyUserAccount
{
     companyAuthId!: number;
     companyAuth!: CompanyAuth;

     userAccountId!: number;
     userAccount!: UserAccountAuthDto;

     LinkedOn!: Date;
}

